### PR TITLE
Adding more examples to example docs

### PR DESCRIPTION
**Pull request by adamgordonbell**
Share larger examples and community-generated examples that are not included in this repository. 

cc @vladaionescu

### CODE REVIEW

File: docs/examples/examples.md [hunk 1]
```diff
@@ -82,6 +82,15 @@ Please note that these examples, although similar, are distinct from the ones us
 * [readme](https://github.com/earthly/earthly/tree/main/examples/readme) - some sample code we used in our README.
 * [tests](https://github.com/earthly/earthly/tree/main/tests) - a suite of tests Earthly uses to ensure that its features are working correctly.
 
+### Larger Examples And Community Examples
+
+* [Earthly Rust, GoLang, NodeJS and GitHubActions Example](https://github.com/earthly/earthly-vs-gha)
```
Comment (vladaionescu) [inline docs/examples/examples.md:87]:
```suggestion
* [Earthly, Rust, GoLang, NodeJS and GitHub Actions Example](https://github.com/earthly/earthly-vs-gha)
```

File: examples/README.md [hunk 2]
```diff
@@ -61,3 +61,12 @@ Please note that these examples, although similar, are distinct from the ones us
 
 * [readme](./readme) - some sample code we used in our README.
 * [tests](./tests) - a suite of tests Earthly uses to ensure that its features are working correctly.
+
+### Larger Examples And Community Examples
+
+* [Earthly Rust, GoLang, NodeJS and GitHubActions Example](https://github.com/earthly/earthly-vs-gha)
```
Comment (vladaionescu) [inline examples/README.md:67]:
```suggestion
* [Earthly, Rust, GoLang, NodeJS and GitHub Actions Example](https://github.com/earthly/earthly-vs-gha)
```

File: docs/examples/examples.md [hunk 3]
```diff
@@ -82,6 +82,15 @@ Please note that these examples, although similar, are distinct from the ones us
...
+* [Cloud Services In GoLang](https://github.com/adamgordonbell/cloudservices)
+* [Earthlfile workshop Repo](https://github.com/earthly/workshop-2023-09-18)
```
Comment (vladaionescu) [inline docs/examples/examples.md:89]:
```suggestion
* [Earthfile workshop Repo](https://github.com/earthly/workshop-2023-09-18)
```

File: examples/README.md [hunk 4]
```diff
@@ -61,3 +61,12 @@ Please note that these examples, although similar, are distinct from the ones us
...
+* [Cloud Services In GoLang](https://github.com/adamgordonbell/cloudservices)
+* [Earthlfile workshop Repo](https://github.com/earthly/workshop-2023-09-18)
```
Comment (vladaionescu) [inline examples/README.md:69]:
```suggestion
* [Earthfile workshop Repo](https://github.com/earthly/workshop-2023-09-18)
```

File: docs/examples/examples.md [hunk 5]
```diff
@@ -82,6 +82,15 @@ Please note that these examples, although similar, are distinct from the ones us
 * [readme](https://github.com/earthly/earthly/tree/main/examples/readme) - some sample code we used in our README.
 * [tests](https://github.com/earthly/earthly/tree/main/tests) - a suite of tests Earthly uses to ensure that its features are working correctly.
 
+### Larger Examples And Community Examples
+
+* [Earthly Rust, GoLang, NodeJS and GitHubActions Example](https://github.com/earthly/earthly-vs-gha)
+* [Cloud Services In GoLang](https://github.com/adamgordonbell/cloudservices)
```
Comment (vladaionescu) [inline docs/examples/examples.md:88]: Should we transfer this example under the earthly GitHub org?

File: docs/examples/examples.md [hunk 6]
```diff
@@ -82,6 +82,15 @@ Please note that these examples, although similar, are distinct from the ones us
 * [readme](https://github.com/earthly/earthly/tree/main/examples/readme) - some sample code we used in our README.
 * [tests](https://github.com/earthly/earthly/tree/main/tests) - a suite of tests Earthly uses to ensure that its features are working correctly.
 
+### Larger Examples And Community Examples
+
+* [Earthly, Rust, GoLang, NodeJS and GitHub Actions Example](https://github.com/earthly/earthly-vs-gha)
+* [Cloud Services In GoLang](https://github.com/earthly/cloud_services_example)
```
Comment (vladaionescu) [inline docs/examples/examples.md:88]: Nit: We generally use kebab-case for repo names.

**Review by vladaionescu — APPROVED** [review]

File: docs/examples/examples.md (hunk 6 above)
Comment (adamgordonbell) [reply docs/examples/examples.md]: Fixed

**Review by alexcb — APPROVED** [review]

**Pull request merged**